bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -0,0 +1,412 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages, injectIntl, FormattedDate } from 'react-intl';
+import Dropzone from 'react-dropzone';
+import update from 'immutability-helper';
+import cx from 'classnames';
+
+import ModalFullscreen from '/imports/ui/components/modal/fullscreen/component';
+import Icon from '/imports/ui/components/icon/component';
+import ButtonBase from '/imports/ui/components/button/base/component';
+import Checkbox from '/imports/ui/components/checkbox/component';
+import styles from './styles.scss';
+
+const DEFAULT_FILENAME = 'default.pdf';
+
+const propTypes = {
+  defaultFileName: PropTypes.string.isRequired,
+  fileSizeMin: PropTypes.number.isRequired,
+  fileSizeMax: PropTypes.number.isRequired,
+  handleSave: PropTypes.func.isRequired,
+  fileValidMimeTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+  presentations: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    filename: PropTypes.string.isRequired,
+    isCurrent: PropTypes.bool.isRequired,
+    conversion: PropTypes.object,
+    upload: PropTypes.object,
+  })).isRequired,
+};
+
+const defaultProps = {
+  defaultFileName: 'default.pdf',
```

Comment:
Please pick the name of the default presentation from a config file (there was another spot above)